packages/nimble-components/src/list-option/index.ts

```diff
@@ -66,6 +66,13 @@ export class ListOption extends FoundationListboxOption {
         super.connectedCallback();
         const owner = this.getListOptionOwner();
         owner?.registerOption(this);
+        const observer = new MutationObserver(_ => {
+            this.$emit('display-changed');
+        });
+        observer.observe(this, {
```

Comment:
Should you call `observer.disconnect()` in the `disconnectedCallback()`?

Reply:
I've altered the implementation according to @rajsite's feedback, and I have included calling `disconnect` in the `disconnectedCallback` method for the `Select`.